### ISSUE TITLE
gnt-network connect --help incorrect "Usage"

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#952]
Originally reported of Google Code with ID 896.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

<b>What distribution are you using?</b>
# gnt-cluster --version
gnt-cluster (ganeti v2.11.3) 2.11.3
# gnt-cluster version
Software version: 2.11.3
Internode protocol: 2110000
Configuration format: 2110000
OS api version: 20
Export interface: 0
VCS version: (ganeti) version v2.11.3
# hspace --version
hspace (ganeti) version v2.11.3
compiled with ghc 7.4
running on linux x86_64
# cat /etc/debian_version
7.6
# apt-cache policy ganeti
ganeti:
  Installed: 2.11.3-2~bpo70+1
  Candidate: 2.11.3-2~bpo70+1
  Package pin: 2.11.3-2~bpo70+1
  Version table:
 *** 2.11.3-2~bpo70+1 990
        100 http://debian.xxxxxxxx.de/debian/ wheezy-backports/main amd64 Packages
        100 /var/lib/dpkg/status
     2.10.5-1~bpo70+1 990
        100 http://debian.xxxxxxxx.de/debian/ wheezy-backports/main amd64 Packages
     2.9.5-1~bpo70+1 990
        100 http://debian.xxxxxxxx.de/debian/ wheezy-backports/main amd64 Packages


<b>What steps will reproduce the problem?</b>
gnt-network connect --help


<b>What is the expected output? What do you see instead?</b>
  Usage
  =====
-   gnt-network connect <network_name> <mode> <link> [<node_group>...]
+   gnt-network connect [...] <network_name> [<node_group>, ...]
  
  Map a given network to the specified node group with given mode and link
  (netparams)
[...]


<b>Please provide any additional information below.</b>

```
Originally added on 2014-07-24 09:01:55 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-07-24 12:15:48 +0000 UTC.

Added Labels: Type-Documentation
Changed State: Accepted
Added to Milestone: Release2.11

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 07b70f64358cfe6135c322e7a906a76589001f8a
Author: Klaus Aehlig <aehlig@google.com>
Date:   Thu Jul 24 14:39:29 2014 +0200

    Update usage string for 'gnt-network connect'
    
    Commit 2243b133 removed the old, positional, passing of mode
    and link arguments. Adapt the documentation string accordingly.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

```
Originally added on 2014-07-24 13:06:18 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.10
Removed from Milestone: Release2.11

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.11.6
```
Originally added on 2014-09-22 15:56:40 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.11.6
```
Originally added on 2014-09-22 15:57:10 +0000 UTC.

Changed State: Released
